dev-docs/feature-format-matrix/qmd-files/layout/fig-align/toplevel-id-caption.qmd

```diff
@@ -0,0 +1,33 @@
+---
+format:
+  html:
+    quality: 1
+  pdf:
+    quality: 1
+  typst:
+    quality: 0
+  dashboard:
+    quality: -1
+  docx:
+    quality: 1
+  pptx:
+    quality: -1
+keep-typ: true
+_quarto:
+  tests:
+# couldn't get either of these to work
+#    pdf:
+#      ensureFileRegexMatches:
+#      -
+#        - "\\\\hfill\\\\includegraphics\\{100.png\\}"
+#        - '\hfill\includegraphics\{100.png\}'
+#      - []
+    typst:
+      ensureTypstFileRegexMatches:
```

Comment:
As I explained in my other answer you need to use `latex`
```suggestion
    latex:
      ensureFileRegexMatches:
      -
        - "\\\\hfill\\\\includegraphics\\{100.png\\}"
        - '\hfill\includegraphics\{100.png\}'
      - []
    typst:
      ensureTypstFileRegexMatches:
```

You can add `pdf: null` also if you want to test pdf rendering with LaTeX, and not just the output